src/internal/metric/hook.ts

```diff
@@ -163,7 +160,7 @@ export const histogram = (key: MetricKey.MetricKey.Histogram): MetricHook.Metric
     }
   }
 
-  const getBuckets = (): Chunk.Chunk<readonly [number, number]> => {
+  const getBuckets = (): ReadonlyArray<readonly [number, number]> => {
```

Comment:
I believe @fubhy had implemented a fix here https://github.com/Effect-TS/effect/pull/1731. We could potentially incorporate into this PR since the return type is changing?